datalad/metadata/metadata.py

```diff
@@ -357,7 +444,7 @@ def __call__(
                     db[k] = v
                 for k, v in add.items():
                     db[k] = sorted(unique(
-                        db.get(k, []) + v))
+                        db.get(k, []) + list(v)))
```

Comment:
Couldn't v be a string, this coffee working without failure but adding bunch of characters? (Blind question - do not see the rest of the coffee ATM) there is `assure_list`